andela/bp-esa-frontend
# #167294842:fix noko project tag not showing

#### What does this PR do?
- This PR fixes the issue of missing project tag under Noko model.

#### Description of Task to be completed?
-  Currently, the `ESA-API` is not returning the `project-name` and all `projects-IDs` are null. This PR partly corrects that issue by filling the project fields with `N/A` in the Noko model at the front-end just as shown in the screenshot below.  

#### How should this be manually tested?
- Clone the repository
- Run `yarn` in the frontend to install new dependencies.
- Run `yarn start` 
- Click the details icon in the right top corner, a model will appear.
- Select the Noko on the model.

#### Any background context you want to provide?
The `Noko project tag field` was not being populated by any values, this was due to back-end issues with the `Noko-API`.

#### What are the relevant pivotal tracker stories?
[#167294842](https://www.pivotaltracker.com/story/show/167294842)

#### Postman Documentation
N/A
#### Screenshots (If applicable)
<img width="498" alt="Screen Shot 2019-08-16 at 16 15 34" src="https://user-images.githubusercontent.com/22595454/63171375-23a78100-c044-11e9-9a3c-33c48b9e4171.png">
